compiler/quilt/test/test_import.py

```diff
@@ -41,6 +41,13 @@ def test_imports(self):
         assert set(dataframes._group_keys()) == set()
         assert set(dataframes._data_keys()) == {'csv', 'nulls'}
 
+        # this should contain elements
+        for node in package:
+            assert node
```

Comment:
This doesn't actually check that package contains elements; it won't run at all if the iterator doesn't return anything.

Do something like `assert list(package)` instead.